docgen/src/index.md

```diff
@@ -2,4 +2,4 @@
 title: react-instantsearch
 layout: index.pug
 ---
-<!-- the content is in another template -->
+<!-- the content is in layouts/index.pug -->
```

Comment:
That was meant to be a joke in reference to Mario (but indeed it's hard to get :P)